docs/arcaflow/workflows/expressions.md

```diff
@@ -39,7 +41,51 @@ For example, to have the following text represented in a single string:
 
 You would need the expression `"test\ntest2/\\"`
 
-### Integer
+#### String-expressions in YAML
+
+If an expression has a string literal in it, then you must account for it in the YAML.
+
+For example, to include a double-quoted string in an expression, you must either add single quotes around expression, or use block flow scalars. A single apostrophe within an expression needs to be added twice in a row to count as one while in an inlined string.
+
+Here is an example of the following value represented in a few of the various ways:
+> just a string with sub-quotes "hello" and an apostrophe '
+
+Inlined:
+```
+some_value: !expr '"just a string with sub-quotes \"hello\" and an apostrophe \'' "'
+```
+
+With Block Flow Scalar:
+```
+some_value_1: !expr |-
+  'just a string with sub-quotes "hello" and an apostrophe \' '
+some_value_2: !expr |-
+  "just a string with sub-quotes \"hello\" and an apostrophe ' "
+```
```

Comment:
Again, additional explanation is in order:
- Explain what the vertical bar does and why is it followed by a hyphen;
- Point out that the string is passed verbatim to Arca;
- In `some_value_1`:  explain that the embedded apostrophe is escaped to avoid causing the string to end prematurely.
- In `some_value_2`:  explain that the embedded double-quotes are escaped to avoid causing the string to end prematurely, and point out that the apostrophe does not need to be escaped.